src/Timers.cpp

```diff
@@ -210,6 +225,218 @@ bool Timers::FindTagInTimerTags(std::string tag, std::string tags)
     return (regex_match(tags, regex));
 }
 
+std::string Timers::ConvertToAutoTimerTag(std::string tag)
+{
+    std::regex regex (" ");
+    std::string replaceWith = "_";
+
+    return regex_replace(tag, regex, replaceWith);
+}
+
+std::vector<AutoTimer> Timers::LoadAutoTimers()
+{
+  std::vector<AutoTimer> autoTimers;
+
+  std::string url; 
+  url = StringUtils::Format("%s%s", vuData.getConnectionURL().c_str(), "autotimer"); 
+
+  std::string strXML;
+  strXML = vuData.GetHttpXML(url);
```

Comment:
You could simplify this and other places slightly by declaring and initializing vars in one go:

`std::string strXML= vuData.GetHttpXML(url);`

And if the var is not modified afterwards a `const` in front of var's type is good style (and can help the compiler to optimize code).